src/lib/helpers.ts

```diff
@@ -1 +1,7 @@
-export const isBrowser = () => typeof window !== 'undefined'
+export const getGlobalFetch = (): typeof fetch | undefined => {
```

Comment:
Optional: I think `=> typeof fetch === 'undefined' ? undefined : fetch` is more readable in this case.